decidim-comments/app/frontend/comments/comments.component.jsx

```diff
@@ -95,6 +95,7 @@ const CommentsWithData = graphql(gql`
   ${commentsQuery}
   ${CommentThread.fragments.comment}
 `, {
+  options: { pollInterval: 15000 },
```

Comment:
That's the...

![](http://static1.squarespace.com/static/554c70f9e4b019d5490b6b99/t/5554a392e4b0f33dbaf754ee/1474746356598/?format=150w)

(sorry)